modules/artifact/src/main/java/org/mule/runtime/module/artifact/serializer/CustomJavaSerializationProtocol.java

```diff
@@ -44,6 +47,13 @@ public CustomJavaSerializationProtocol(ClassLoaderRepository classLoaderReposito
    */
   @Override
   protected byte[] doSerialize(Object object) throws Exception {
+    //TODO: MULE-11939
```

Comment:
That is the issue that is referenced on both Java and Kryo serializers....  @marianogonzalez can you clarify if the issue status is correct or there is still things to do for it as pointed on those TODOs?